ui/job-view/pushes/Push.jsx

```diff
@@ -69,15 +69,22 @@ class Push extends React.PureComponent {
     window.removeEventListener(thEvents.applyNewJobs, this.handleApplyNewJobs);
     window.removeEventListener('hashchange', this.handleUrlChanges);
   }
-
+  
   getJobCount(jobList) {
+    function filterCommit(job) {
+      if (job.failure_classification_id == 2) {
+        return true
+      }
+    }    
+    const filteredByCommit = jobList.filter(filterCommit);
```

Comment:
Now that you have created this value, you need to return it from this function with the rest of what is returned.  One way to do that would be to return it as part of the ``initial`` value of the ``reduce`` function below.